app/services/users/default_categories_creator.rb

```diff
@@ -0,0 +1,17 @@
+class Users::DefaultCategoriesCreator < Struct.new(:user)
+  def call
+    default_categories_list.map { |title| create_category(title) }
+  end
+
+  private
+
+  def create_category(title)
```

Comment:
@marderer private methods with arguments say that a service is overloaded and you have to pass arguments to private methods

what do you think about:
```ruby
def call
  default_categories_list.map do |name|
    user_categories.create(name: name, amount: 0)
  end
end
```
?

Reply:
@AleksSenkou Maybe better this method make private? And call in the method `call`? 

``` ruby
  def call
    create_default_categories
  end

  private

  def create_default_categories
    default_categories_list.map do |name|
      user_categories.create(name: name, amount: 0)
    end
  end

```

Reply:
@marderer hah, here is a play with names, when you call just one method in `call` says that we need just one method called `#create_default_categories` instead of `#call` 😄 

So, we can go with one public method `#create_default_categories`